_layouts/contact.html

```diff
@@ -1,10 +1,15 @@
 {% include header.html %}
 <section id="main" class="container">
   <header>
-    {{content}}
+        {{content}}
+        <hr>
+        <h4><b>{{page.street}}</b></h4>
+        <h1><b>{% if {{page.lang}} == "FR" %}Téléphone:{% else %}Phone:{% endif %} </b> {{page.phone}}</h1>
```

Comment:
The following if on the two next line could be removed for only one. This would reduce the processing of generating the page